src/js/Parser.js

```diff
@@ -0,0 +1,92 @@
+import { Station } from './Station';
+import { SubwayLine } from './SubwayLine';
+import { Subway } from './Subway';
+
+export function parseToSubway(data) {
```

Comment:
why name is parseToSubway? parseSubwayData or just parse is ok. parseToSubway isn't like English phrase